exp-fashion-mnist/hyperparams.json

```diff
@@ -1,35 +1,31 @@
 {
     "adagrad": {
-        "initial_batch_size": 256,
-        "lr": 0.0056328758,
-        "weight_decay": 0.0003
-    },
-    "geodamp": {
-        "dampingdelay": 6,
-        "dampingfactor": 2.5,
-        "initial_batch_size": 64,
-        "lr": 0.0056328758,
-        "max_batch_size": 4096,
+        "dwell": 20,
+        "initial_batch_size": 128,
+        "lr": 0.005,
         "momentum": 0.9,
-        "nesterov": 1,
-        "weight_decay": 0.0003
+        "weight_decay": 0.003
     },
-    "geodamplr": {
-        "dampingdelay": 6,
-        "dampingfactor": 2.5,
-        "initial_batch_size": 64,
-        "lr": 0.0056328758,
-        "max_batch_size": 64,
+    "geodamp": {
+        "dampingdelay": 10,
+        "dampingfactor": 1.2192312516491106,
+        "dwell": 20,
+        "initial_batch_size": 256,
+        "lr": 0.005,
+        "max_batch_size": 512,
```

Comment:
I think we should talk about `max_batch_size`. Increasing for 40 epochs/only by a factor of 2 seems small.

Reply:
```suggestion
        "max_batch_size": 1024,
```